showcase/app/components/page-carbonization/components/badge.gts

```diff
@@ -0,0 +1,352 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+import { pageTitle } from 'ember-page-title';
+import { array } from '@ember/helper';
+
+import ShwTextH1 from 'showcase/components/shw/text/h1';
+import ShwTextH2 from 'showcase/components/shw/text/h2';
+import ShwFlex from 'showcase/components/shw/flex';
+import ShwDivider from 'showcase/components/shw/divider';
+import ShwCarbonizationComparisonGrid from 'showcase/components/shw/carbonization/comparison-grid';
+
+import {
+  HdsBadge,
+  HdsIcon,
+} from '@hashicorp/design-system-components/components';
+import {
+  SIZES,
+  TYPES,
+} from '@hashicorp/design-system-components/components/hds/badge/index';
+
+const BadgeCarbonization: TemplateOnlyComponent = <template>
+  {{pageTitle "Badge - Carbonization"}}
+
+  <ShwTextH1>Badge - Carbonization</ShwTextH1>
+
+  <section>
+
+    <ShwTextH2>Content</ShwTextH2>
+
+    <ShwCarbonizationComparisonGrid>
+      <:theming>
+        <HdsBadge @text="Only text" />
+      </:theming>
+      <:reference>
+        <cds-tag>Lorem ipsum</cds-tag>
```

Comment:
At first glance, the `cds-tag` name seemed like a generic element/component name to me (like an HTML "tag") but I see that they are calling they call their "badge" a "tag"... :P